managerInterfaces.go

```diff
@@ -8,7 +8,7 @@ import (
 type tokenManager interface {
 	getBearerToken() string
 	invalidateToken() error
-	getToken() *oauthToken
+	getToken() *OauthToken
```

Comment:
Yes. In fact my docker test is in a separate package. I write a simple dummy token auth server to make apid happy.